ranger/data/scope.sh

```diff
@@ -40,7 +40,6 @@ HIGHLIGHT_STYLE=${HIGHLIGHT_STYLE:-pablo}
 HIGHLIGHT_OPTIONS="--replace-tabs=${HIGHLIGHT_TABWIDTH} --style=${HIGHLIGHT_STYLE} ${HIGHLIGHT_OPTIONS:-}"
 PYGMENTIZE_STYLE=${PYGMENTIZE_STYLE:-autumn}
 
-
```

Comment:
I think we want these double whitespaces back, here and at l266. Otherwise this looks ok for now.

I have a PR in the works to add shellcheck to our CI, that might flag some issues with your commands, things like not putting ` -- ` in front of the `FILE_PATH` arguments so they won't get parsed as options/flags. So I'm gonna hold off on merging this until after.

Reply:
Last commit:
- add `--` to all the apps I had added (after testing that it works)
- re-add empty lines
- add fallback pandoc support to html and json